main/src/ca/uwaterloo/flix/language/phase/constraintgeneration/TypeContext.scala

```diff
@@ -122,15 +123,17 @@ class TypeContext {
     *   tpe1 ~ tpe3
     * }}}
     */
-  def unifyType3(tpe1: Type, tpe2: Type, tpe3: Type, loc: SourceLocation): Unit = {
```

Comment:
Overloads make me sad, but ok.

Reply:
I'm just being consistent. We could get rid of them but then it must be everywhere